test/jdk/javax/management/remote/mandatory/connection/DefaultAgentFilterTest.java

```diff
@@ -282,6 +291,28 @@ public static void main(String[] args) throws Exception {
             System.out.println("----\tTest FAILED !!");
             throw ex;
         }
+        try {
+            // Use default filter, should fail with: java.io.InvalidClassException: filter status: REJECTED
+            testDefaultAgent(null /* no properties file */);
+            throw new RuntimeException("---" + DefaultAgentFilterTest.class.getName() + " - No exception reported");
+        } catch (Exception ex) {
+            System.out.println(ex);
+            if (ex instanceof InvalidClassException) {
+                System.out.println("----\tTest PASSED: expected InvalidClassException received.");
+            } else {
+                System.out.println("----\tTest FAILED: unexpected Exception");
+                throw ex;
+            }
+        }
+        try {
+            // Add custom filter on command-line.
+            testDefaultAgent(null, "-Dcom.sun.management.jmxremote.serial.filter.pattern=\"java.lang.*;java.math.BigInteger;java.math.BigDecimal;java.util.*;javax.management.openmbean.*;javax.management.ObjectName;java.rmi.MarshalledObject;javax.security.auth.Subject;DefaultAgentFilterTest$MyTestObject;!*\"");
```

Comment:
Long line. Can we use string concatenation to break it?

Reply:
Yes, done!